### PR TITLE
[DOC-4789] Documentation README.md review.

### PR DESCRIPTION
**Pull request by elatt**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Minor updates to new Flask extension model template.

## Rationale
Final documentation review before closing out the project.

### CODE REVIEW

File: model_templates/flask_extension_httpauth/README.md [hunk 1]
```diff
@@ -1,22 +1,15 @@
...
-Note: it is **not** necessary (nor recommended) to add authentication to custom models that are created in DataRobot MLOps.
-This example is simply to demonstration the flexibility of the `custom_flask.py` hook.
+To extend the HTTP server, first create a new custom model with the files in this directory and the Python Drop-In Environment.
 
-## Instructions
-Create a new custom model with these files and use the Python Drop-In Environment with it.
+**Important**: You can only extend the web server when running it _without_ the `--production` flag (or `PRODUCTION=1` environment variable).
 
-**Important:** extending the web server is only available when running **without** the `--production` flag (or `PRODUCTION=1` environment variable).
+To run the model locally with 'drum', use the following command with paths relative to `./datarobot-user-models`:
```
Comment (elatt) [inline model_templates/flask_extension_httpauth/README.md:11]:
```suggestion
To run this example locally with, use the following command with paths relative to `./datarobot-user-models`:
```

File: model_templates/flask_extension_httpauth/README.md [hunk 2]
```diff
@@ -1,22 +1,15 @@
...
-when drum is in server-mode. To demonstrate one potential usecase, the `custom_flask.py` file in
-this model directory will extend the HTTP server to require a specific [Bearer Token](https://swagger.io/docs/specification/authentication/bearer-authentication/)
-when making any requests to it.
+This sample illustrates how you can add third-party or custom Flask extensions when DRUM is in server mode. To demonstrate one potential use case, the `custom_flask.py` file in this model directory extends the HTTP server to require a specific [Bearer Token](https://swagger.io/docs/specification/authentication/bearer-authentication/) when making any requests to it. For completeness, this directory also includes all model-related files from the [Python Sklearn Inference Model Template](../python3_sklearn/).
 
-For completeness, we also include all the model related files from the [Python Sklearn Inference Model Template](../python3_sklearn/).
+**Note**: It is _not_ necessary (or recommended) to add authentication to custom models created in DataRobot MLOps. This example is simply to demonstrate the flexibility of the `custom_flask.py` hook.
```
Comment (elatt) [inline model_templates/flask_extension_httpauth/README.md:5]:
```suggestion
**Note**: It is _not_ necessary (or recommended) to add authentication to custom models that will be deployed in DataRobot MLOps -- the platform will layer its standard API authentication on top of your custom model for you. This example is simply to demonstrate the flexibility of the `custom_flask.py` hook or to show how you can add authentication to a 'drum' model that is served externally.
```

Comment (elatt) [reply model_templates/flask_extension_httpauth/README.md:5]: @n-aylward do you think this is too wordy?

Comment (n-aylward) [reply model_templates/flask_extension_httpauth/README.md:5]: Works for me! I just replaced `--` with `&mdash;`.

File: model_templates/flask_extension_httpauth/README.md [hunk 3]
```diff
@@ -1,22 +1,15 @@
...
-Note: it is **not** necessary (nor recommended) to add authentication to custom models that are created in DataRobot MLOps.
-This example is simply to demonstration the flexibility of the `custom_flask.py` hook.
+To extend the HTTP server, first create a new custom model with the files in this directory and the Python Drop-In Environment.
 
-## Instructions
-Create a new custom model with these files and use the Python Drop-In Environment with it.
+**Important**: You can only extend the web server when running it _without_ the `--production` flag (or `PRODUCTION=1` environment variable).
 
-**Important:** extending the web server is only available when running **without** the `--production` flag (or `PRODUCTION=1` environment variable).
+To run this example locally with, use the following command with paths relative to `./datarobot-user-models`:
```
Comment (n-aylward) [inline model_templates/flask_extension_httpauth/README.md:11]:
```suggestion
To run this example locally, use the following command with paths relative to `./datarobot-user-models`:
```

**Review by n-aylward — APPROVED** [review]

**Comment by elatt** [comment]
jenkins run test_integration_general test_integration_r_environment

**Comment by elatt** [comment]
jenkins run test_integration_general

File: model_templates/flask_extension_httpauth/README.md [hunk 4]
```diff
@@ -1,22 +1,15 @@
...
 
-For completeness, we also include all the model related files from the [Python Sklearn Inference Model Template](../python3_sklearn/).
+**Note**: It is _not_ necessary (or recommended) to add authentication to custom models that will be deployed in DataRobot MLOps — the platform will layer its standard API authentication on top of your custom model for you. This example is simply to demonstrate the flexibility of the `custom_flask.py` hook or to show how you can add authentication to a 'drum' model that is served externally.
 
-Note: it is **not** necessary (nor recommended) to add authentication to custom models that are created in DataRobot MLOps.
-This example is simply to demonstration the flexibility of the `custom_flask.py` hook.
+To extend the HTTP server, first create a new custom model with the files in this directory and the Python Drop-In Environment.
```
Comment (zohar-mizrahi) [inline model_templates/flask_extension_httpauth/README.md:7]: Is the "Python Drop-In Environment" required to be in the "new custom model"? Otherwise, consider rephrasing.

File: model_templates/flask_extension_httpauth/README.md [hunk 5]
```diff
@@ -1,22 +1,15 @@
 ## Extending Web Server Behavior
 
-This sample is meant to illustrate how you can add thirdparty or your own custom Flask extensions
-when drum is in server-mode. To demonstrate one potential usecase, the `custom_flask.py` file in
-this model directory will extend the HTTP server to require a specific [Bearer Token](https://swagger.io/docs/specification/authentication/bearer-authentication/)
-when making any requests to it.
+This sample illustrates how you can add third-party or custom Flask extensions when DRUM is in server mode. To demonstrate one potential use case, the `custom_flask.py` file in this model directory extends the HTTP server to require a specific [Bearer Token](https://swagger.io/docs/specification/authentication/bearer-authentication/) when making any requests to it. For completeness, this directory also includes all model-related files from the [Python Sklearn Inference Model Template](../python3_sklearn/).
```
[the remark this answers is not in the document]
Comment (elatt) [reply model_templates/flask_extension_httpauth/README.md:3]: @n-aylward is there a reason you get rid of the newlines? Is that a policy in the docs repo -- what advantage does it provide?

[the remark this answers is not in the document]
Comment (n-aylward) [reply model_templates/flask_extension_httpauth/README.md:3]: Hi @elatt, nope! Feel free to adjust as needed. As mentioned in my comment on the Jira ticket, I just didn't pay any mind to line length while editing. I didn't know what the guidelines were, so I consolidated the lines to edit the content more easily. I'll keep the 80 to 100 limit in mind next time.

**Comment by elatt** [comment]
jenkins run test_integration_general test_integration_java_environment test_integration_r_environment

**Review by yakov-g — APPROVED** [review]

**Comment by yakov-g** [comment]
jenkins run test_integration_general test_integration_r_environment

**Comment by yakov-g** [comment]
jenkins run test_integration_general

**Comment by yakov-g** [comment]
jenkins run test_integration_general
